pkg/filesystem/directory.go

```diff
@@ -112,6 +112,12 @@ type Directory interface {
 	// Function that base types may use to implement calls that
 	// require double dispatching, such as hardlinking and renaming.
 	Apply(arg interface{}) error
+
+	// Mount and Unmount.
+	// This uses `Mountat` functionality, but not `Unmountat`.
+	// see https://github.com/buildbarn/bb-remote-execution/issues/115 for information.
```

Comment:
I'd remove this comment. This is an implementation detail of the Linux/UNIX version of this method, but not a general requirement/property of how this interface is implemented.